### PR TITLE
Update the readme and fix spelling

### PR DESCRIPTION
**Pull request by da-kami**
Note: Removes the Metamask privacy mode section because it will be
automatically checked with a popup from Metamask.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ If there are no instructions for your system (Mac/Ubuntu) then there is nothing
 ## Do a Swap!
 For Ethereum actions: use metamask. Fund only ONCE.
 
-For Bitcoin actions, need to use commandline:
+For Bitcoin actions you need to use the commandline:
 - `alias btc-cli='docker exec -it bitcoind_1 bitcoin-cli -regtest -rpcport=18443 -rpcuser=bitcoin -rpcpassword=54pLR_f7-G6is32LP-7nbhzZSbJs_2zSATtZV_r05yg='`
```
Comment (thomaseizinger) [inline README.md]: This is sufficient.

```suggestion
- `alias btc-cli='docker exec bitcoind_1 bitcoin-cli -regtest'`
```

**Review by thomaseizinger — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2,27 +2,29 @@
 
 Welcome to the workshop!
 
-If there are no instructions for your system (Mac/Ubuntu) then there is nothing to be done.
+The workshop was tested for Ubuntu and Mac only. If there are no instructions for your system then unfortunately there is nothing to be done.
```
Comment (D4nte) [inline README.md]:
```suggestion
In the instructions below, if nothing is specified for your system (Mac/Ubuntu) then you skip the step.
```

File: README.md [hunk 3]
```diff
@@ -2,27 +2,29 @@
...
 Welcome to the workshop!
 
-If there are no instructions for your system (Mac/Ubuntu) then there is nothing to be done.
+In the instructions below, if nothing is specified for your system (Mac/Ubuntu) then you skip the step.
 
 # Slack channel
 
 1. Go to #coblox-workshop
-2. The url of the repo in the topic, clone it: `git clone https://github.com/coblox/2019-genesis-workshop`
+2. The url of the repo in the slack-channel topic, clone it: `git clone https://github.com/coblox/2019-genesis-workshop`
```
Comment (D4nte) [inline README.md]:
```suggestion
2. The url of the repo in the Slack channel topic, clone it: `git clone https://github.com/coblox/2019-genesis-workshop`
```

File: README.md [hunk 4]
```diff
@@ -33,30 +35,28 @@ If there are no instructions for your system (Mac/Ubuntu) then there is nothing
    - `cargo install --path application/btsieve`
 
 ## Bitcoin setup
-1. Go back to `2019-genesis-workshop` folder
-2. In `docker-compose.yml`, replace `BITCOIN_NODE_IP_PORT` with Bitcoin Node IP:PORT from Slack topic 
-3. Start Bitcoin node from the workshop folder: `docker-compose up`
+1. Navigate to the `2019-genesis-workshop` folder
+2. In `docker-compose.yml`, replace `BITCOIN_NODE_IP_PORT` with the Bitcoin Node IP:PORT from the slack-channel topic
```
Comment (D4nte) [inline README.md]:
```suggestion
2. In `docker-compose.yml`, replace `BITCOIN_NODE_IP_PORT` with the Bitcoin Node IP:PORT from the Slack channel topic 
```

File: README.md [hunk 5]
```diff
@@ -33,30 +35,28 @@ If there are no instructions for your system (Mac/Ubuntu) then there is nothing
...
-2. Setup Metamask by following the wizard (new wallet). Yes you have to do the seed phrase thing.
+2. Setup Metamask by following the wizard (new wallet). Yes, you have to do the seed phrase thing.
 3. Connect metamask to our Ethereum node:
   - Choose "Custom RPC"
+    
     ![connect Ethereum](./img/eth_connect.png)
     
   - Network Name: "CoBloX Test"
-  - New RPC URL: Ethereum URL from Slack topic
+  - New RPC URL: Ethereum URL from the slack-channel topic
```
Comment (D4nte) [inline README.md]:
```suggestion
  - New RPC URL: Ethereum URL from the Slack channel topic
```

**Review by D4nte — APPROVED** [review]

**Pull request merged**